Mathlib/MeasureTheory/Decomposition/SignedLebesgue.lean

```diff
@@ -185,13 +185,11 @@ theorem rnDeriv_def (s : SignedMeasure α) (μ : Measure α) : rnDeriv s μ = fu
       (s.toJordanDecomposition.negPart.rnDeriv μ x).toReal :=
   rfl
 
-attribute [eqns rnDeriv_def] rnDeriv
```

Comment:
The porting note above probably should be updated with a sentence about avoiding `@[eqns]`